### PR TITLE
Bare Metal Forms and Helpers Project

### PR DESCRIPTION
**Pull request by Abhigyan001**
In this project, a form is created using Rails helpers form_tag and form_for.

This project includes the following features :

- Create users once the username, email, and password are entered on the form.
- Validates if the input fields have correct inputs.
- Update the username, email id, and password of existing users.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Abhigyan001 and @khalilhamdii 
### You have done well so far :clap: However, you have a few issues to fix;-
- [ ] Add `rails db:create` and `rails db:migrate` as part of the installation in the readme file
- [ ] Add github actions and install all the required dependencies, [follow this link](https://github.com/microverseinc/linters-config/tree/master/ror) and run the commands `rubocop.` and `npx stylelint "**/*.{css,scss}"`
- [ ] When a user is added, let the return message to be a successful one, not the errors message. also, add edit functionality to work as required
### Make all the changes and submit for another code review. Good luck.
- Happy coding!

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (maelfosso) [inline app/controllers/users_controller.rb:7]: Remove comments please

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path(@user)
```
Comment (maelfosso) [inline app/controllers/users_controller.rb:12]: Not `new_user_path(@user)` but only `new_user_path`

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Thank you for applying the suggested changes on previous reviews.
However, there are some issues

1- As indicated in the previous reviews https://github.com/khalilhamdii/Rails_forms/pull/1#pullrequestreview-512862622, update your read-me with information like `rails db:create`, you can't migrate without creating the database first.

2- Kindly read the comments left below, please.

Once making the suggested changes, submit a new code review request
Thanks

Mael FOSSO

**Review by kitpao — APPROVED** [review]
# 👋 Hello @khalilhamdii and @Abhigyan001 

👏 Great work on this project, you have implemented all previous TSE's requirements and your project is working nicely 🌟 

**_Note:_** All the **[OPTIONAL]** are **not required** in order to approve your project but are personal suggestions to improve your coding experience

- [ ] **OPTIONAL** so your project can be compatible with most ruby versions, in your Gemfile line 4 you can use `ruby '>=2.6.0'`

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me on slack as @kitpao
##### 🌟  If you have feedback about my reviews I would appreciate it if you could leave a recommendation in LinkedIn 💌 or let me know in Slack if you see improvements to be done 😄  

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Pull request merged**